ee/codegen/src/utils/typing.ts

```diff
@@ -1,7 +1,27 @@
+import { isNil } from "lodash";
+
 export function isDefined<TValue>(value: TValue | undefined): value is TValue {
   return value !== undefined;
 }
 
 export function assertUnreachable(_: never): never {
   throw new Error("Didn't expect to get here");
 }
+
+export function isNilOrEmpty<T>(
```

Comment:
[nit] Some quick unit tests for this would be nice

Reply:
I will make a follow up PR for this as a fast follow since the other functions in typing could probably use some unit tests as well